spanner/spannertest/integration_test.go

```diff
@@ -137,6 +151,28 @@ func TestIntegration_SpannerBasics(t *testing.T) {
 	}
 	it.Stop()
 
+	// Try to execute the equivalent of a session pool ping.
```

Comment:
Link to the bug and mention that this used to trigger a panic.

Reply:
Done.